discovery/service/control_center.py

```diff
@@ -260,6 +287,58 @@ def _build_kerberos_properties(self, service_prop: dict) -> tuple:
             }
         return 'all', {}
 
+    def _build_connect_ssl_properties(self, service_props) -> tuple:
+        key1 = "confluent.controlcenter.connect.ssl.key.password"
+        key2 = "confluent.controlcenter.connect.ssl.keystore.location"
+        key3 = "confluent.controlcenter.connect.ssl.keystore.password"
+        key4 = "confluent.controlcenter.connect.ssl.truststore.location"
+        key5 = "confluent.controlcenter.connect.ssl.truststore.password"
+
+        self.mapped_service_properties.add(key1)
+        self.mapped_service_properties.add(key2)
+        self.mapped_service_properties.add(key3)
+        self.mapped_service_properties.add(key4)
+        self.mapped_service_properties.add(key5)
+
+        if key1 in service_props:
+            return 'all', {'kafka_connect_ssl_enabled': True}
+
+    def _build_ksql_ssl_properties(self, service_props) -> tuple:
+        key1 = "confluent.controlcenter.ksql.default.ssl.enabled.protocols"
+        key2 = "confluent.controlcenter.ksql.default.ssl.key.password"
+        key3 = "confluent.controlcenter.ksql.default.ssl.keystore.location"
+        key4 = "confluent.controlcenter.ksql.default.ssl.keystore.password"
+        key5 = "confluent.controlcenter.ksql.default.ssl.protocol"
+        key6 = "confluent.controlcenter.ksql.default.ssl.truststore.location"
+        key7 = "confluent.controlcenter.ksql.default.ssl.truststore.password"
+
+        self.mapped_service_properties.add(key1)
+        self.mapped_service_properties.add(key2)
+        self.mapped_service_properties.add(key3)
+        self.mapped_service_properties.add(key4)
+        self.mapped_service_properties.add(key5)
+        self.mapped_service_properties.add(key6)
+        self.mapped_service_properties.add(key7)
+
+        if key1 in service_props:
+            return 'all', {'ksql_ssl_enabled': True}
+
+    def _build_sr_ssl_properties(self, service_props: dict) -> tuple:
+        key1 = "confluent.controlcenter.schema.registry.sr-cluster.ssl.key.password"
+        key2 = "confluent.controlcenter.schema.registry.sr-cluster.ssl.keystore.location"
+        key3 = "confluent.controlcenter.schema.registry.sr-cluster.ssl.keystore.password"
+        key4 = "confluent.controlcenter.schema.registry.sr-cluster.ssl.truststore.location"
+        key5 = "confluent.controlcenter.schema.registry.sr-cluster.ssl.truststore.password"
+
+        self.mapped_service_properties.add(key1)
+        self.mapped_service_properties.add(key2)
+        self.mapped_service_properties.add(key3)
+        self.mapped_service_properties.add(key4)
+        self.mapped_service_properties.add(key5)
+
+        if key1 in service_props:
+            return 'all', {'ksql_ssl_enabled': True}
```

Comment:
Should this be schema_registry_ssl_enabled?

Reply:
Yes. Thanks !